src/bvals/bvals_interfaces.hpp

```diff
@@ -82,21 +82,12 @@ enum {
 };
 #endif
 
-// identifiers for all 6 faces of a MeshBlock
-enum BoundaryFace {
-  undef = -1,
-  inner_x1 = 0,
-  outer_x1 = 1,
-  inner_x2 = 2,
-  outer_x2 = 3,
-  inner_x3 = 4,
-  outer_x3 = 5
-};
 // TODO(felker): BoundaryFace must be unscoped enum, for now. Its enumerators are used as
 // int to index raw arrays (not ParArrayNDs)--> enumerator vals are explicitly specified
 
 // identifiers for boundary conditions
-enum class BoundaryFlag { block = -1, undef, reflect, outflow, periodic };
+// TODO(JMM): Can BoundaryFlag be removed?
```

Comment:
I ended up moving it to `defs.hpp` to avoid a circular dependency in the headers. That said, that `TODO` is clearly wrong and I will remove it.

Reply:
Oh wait. I was thinking of `BoundaryFace`. This TODO was more of... since there's the function pointer machinery, maybe the flags are extraneous. That said, I don't think they are. I will remove the TODO.